cli/commands/configure.ts

```diff
@@ -12,19 +12,43 @@ export const configureCommand = (): CommandDefinition => ({
 			flags: '-d, --deleteBeforeUpdate',
 			description: `Useful when depending on the parameter having version 1, e.g. for use in CloudFormation`,
 		},
+		{
+			flags: '-X, --deleteParameter',
+			description: 'Deletes the parameter.',
+		},
 	],
 	action: async (
 		scope: any,
 		system: any,
 		property: string,
 		value: string | undefined,
-		{ deleteBeforeUpdate },
+		{ deleteBeforeUpdate, deleteParameters },
 	) => {
+		const ssm = new SSMClient({})
+
+		if (deleteParameters !== undefined) {
```

Comment:
```suggestion
		{ deleteBeforeUpdate, deleteParameter },
	) => {
		const ssm = new SSMClient({})

		if (deleteParameter !== undefined) {
```

This also needs to be renamed to match the option.